pkg/reconciler/ingress/config/istio.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
can we share this with `clusteringress/config/istio.go`

Reply:
Regarding sharing code between clusteringress and ingress controllers.    I though I would keep their codes separate and get rid of the clusteringress package when it is time to remove them.   You are right that we still need to maintain/sync them in the interim.    I will make them share code.

Reply:
Yes, thanks.  I think sharing will make clean a tad more complicated, but will probably protect us in the mean time.